LICENSE

```diff
@@ -1,6 +1,7 @@
 BSD 3-Clause License
 
-Copyright (c) 2017, Association Scientifique pour la Geologie et ses Applications (ASGA). All rights reserved.
```

Comment:
I am not against this change. It makes sense since RINGMesh is open-source and I think that your update is correct. But maybe we should double-check with the ASGA and Guillaume to be sure this is fine.

Reply:
For now, it is simply to try to display the License in the github API

Reply:
I agree with Benjamin. Thus PR should be approved by Guillaume

Reply:
Good point, all right with me.